### DISCUSSION TITLE
How to display date on calendar using DateTime value from database

### DISCUSSION THREAD

**Discussion by jaymichalek** [wojtekmaj/react-calendar#636] [Q&A]
I am very new to React and I found the react-calendar through searching and used it as a date picker for my app. When I save a new entity on my app, it saves the date from react-calendar to my database just fine. But when I am updating the entity, is there a way to show the datetime value on the react-calendar?

**Comment by wojtekmaj** [comment]
Is there a shareable piece of code you're working on? You could use CodeSandbox or similar to build a quick repro for your use case for me to have a look and help you out.

**Reply by jaymichalek** [reply]
@wojtekmaj I was able to solve the issue. The date previously selected is now rendering properly on react-calendar. I was also able to save the new date selected. The wonderful documentation here and the recipes helped a lot. Here is my sandbox for it: https://codesandbox.io/s/agitated-visvesvaraya-688cob?file=/src/UpdatePatient.js
Thank you.
